requirements.txt

```diff
@@ -12,6 +12,7 @@ django-jet==1.0.8
 django-ckeditor==5.6.1
 drf_yasg==1.15.0
 django-crispy-forms==1.7.2
+mysql-connector-python==8.0.18
```

Comment:
Why do you need this add?

Reply:
here https://github.com/Ensembl/ensembl-production-services/blob/new_copy_service/ensembl_dbcopy/api/views.py#L17

Reply:
Fine, I presume you tried to reuse MySQL connection from Django, but anyway, these are tied to settings.DATABASES values